transport/nio/src/test/scripts/org/kaazing/specification/udp/rfc768/additions/align.content/client.rpt

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+connect await BOUND
+        udp://localhost:8080
+connected
+
+write [0x00 0x01 0x02]
```

Comment:
I thought each write on udp would generate a datagram on the wire. Tried with flush but it seems to hang.